Pilot/pilotTools.py

```diff
@@ -214,53 +214,97 @@ def listdir(directory):
     return contents
 
 
-def getFlavour(ceName):
+def getSubmitterInfo(ceName):
+    """
+    Check the environment variables to determine the type of batch system and CE used 
+    to submit the pilot being used and return this information in a tuple.
+    """
 
-    pilotReference = os.environ.get("DIRAC_PILOT_STAMP", "")
-    flavour = "DIRAC"
+    pilotReference = os.environ.get("DIRAC_PILOT_STAMP", "Unknown")
+    # Batch system taking care of the pilot
+    # Might be useful to extract the info to interact with it later on
+    batchSystemType = None
+    batchSystemJobID = None
+    batchSystemParameters = {
+        "BinaryPath": "Unknown",
+        "Host": "Unknown",
+        "InfoPath": "Unknown",
+        "Queue": "Unknown",
+    }
+    # Flavour of the pilot
+    # Inform whether the pilot was sent through SSH+batch system or a CE
+    flavour = "Unknown"
```

Comment:
Why not anymore `flavour = "DIRAC"` ?

Reply:
I don't know, `"Unknown"` seemed more descriptive than `"DIRAC"` to me.
Any reason to name it `"DIRAC"`?

Reply:
Because normally wherever we run through SSH we call the site "DIRAC.name.country"

Reply:
I reverted the line, but I still don't understand the logic here.